netmaster/master/netmaster_test.go

```diff
@@ -296,3 +308,33 @@ func TestVxlanConfigWithLateHostBindings(t *testing.T) {
 
 	verifyKeys(t, keys)
 }
+
+// Tests for https://github.com/contiv/netplugin/issues/214
+func TestVxlanConfigPktTagOutOfRange(t *testing.T) {
+	cfgBytes := []byte(`{
+    "Tenants" : [{
+        "Name"                  : "tenant-one",
+        "DefaultNetType"        : "vxlan",
+        "SubnetPool"            : "11.1.0.0/16",
+        "AllocSubnetLen"        : 24,
+        "Vxlans"                : "2001-3000",
+        "Networks"  : [{
+            "Name"              : "orange",
+            "PktTag"            : 1300
+        }]
+    }]}`)
+
+	initFakeStateDriver(t)
+	defer deinitFakeStateDriver()
+
+	// Create the Tenant and Network
+	applyConfig(t, cfgBytes)
```

Comment:
@vijayvikrant - hmm.. if it doesn't return the error, how do we make sure that in reality it returns error as well (say from netctl/contivctl)?
> As of today create Network will not return an error for this. So, what you are saying is that I check for err nonetheless, so that when we fix it tomorrow, that also gets verified?